github.go

```diff
@@ -16,6 +16,10 @@ type Repositories interface {
 	CreateStatus(owner, repo, ref string, status *github.RepoStatus) (*github.RepoStatus, *github.Response, error)
 }
 
+type Issues interface {
+	AddLabelsToIssue(owner, repo string, number int, labels []string) ([]github.Label, *github.Response, error)
```

Comment:
Maybe rename `number` to `issueNumber` for clarity and consistency with `addLabel(...)` (line 61).